android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -42,59 +56,99 @@ public void setDescription(String description) {
 
     @ReactMethod
     public void setMetadata(ReadableMap metadataMap) throws JSONException {
-        Radar.setMetadata(RNRadarUtils.jsonObjectForMap(metadataMap));
-    }
-
-    @ReactMethod
-    public void setPlacesProvider(String providerStr) {
-        Radar.setPlacesProvider(RNRadarUtils.placesProviderForString(providerStr));
+        Radar.setMetadata(RNRadarUtils.jsonForMap(metadataMap));
     }
 
     @ReactMethod
     public void getPermissionsStatus(Promise promise) {
```

Comment:
seems odd to pass a `Promise` as a param to a function rather than have it as a return type -- I guess if that's the RN way 🤷‍♂

Reply:
Yeah, this is standard for RN modules. Docs: https://reactnative.dev/docs/native-modules-android